frontend/src/slices/chat/chat.handler.ts

```diff
@@ -0,0 +1,35 @@
+import { type ChatMessage, type ChatState } from '#libs/types/types.js';
+
+const LAST = -1;
+const FIRST = 0;
```

Comment:
It's better to separate constants in the corresponding file and also make their names more describable